recipes/pyjion/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "pyjion" %}
+{% set version = "1.2.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/tonybaloney/Pyjion.git
+  git_tag: {{ version }}
+
+
+build:
+  # skip: True  # [py < 310]
+  number: 0
+  script: |
+    git submodule update --init --recursive
+    export CMAKE_MAKE_PROGRAM="$(which ninja)"
+    echo '####################################################################'
+    echo "${CMAKE_MAKE_PROGRAM}"
+    echo '####################################################################'
+    {{ PYTHON }} -m pip install . -vv
```

Comment:
It looks like this is working:

```
####################################################################
$BUILD_PREFIX/bin/ninja
####################################################################
```
...but it still doesn't appear to be being tried:
```
  -- Trying "Unix Makefiles" generator - failure
  --------------------------------------------------------------------------------

  ********************************************************************************
  scikit-build could not get a working generator for your system. Aborting build.
```